system_reports/linux-docker-android-lts.log

```diff
@@ -220,21 +220,21 @@ WARNING: No swap limit support
 
 === Docker pre-cached images ===========
 quay.io/bitriseio/android-ndk-lts - 21.3GB
+quay.io/bitriseio/bitrise-base - 2.11GB
 quay.io/bitriseio/android - 17.8GB
-quay.io/bitriseio/bitrise-base - 2.1GB
 ========================================
 
 
 === System infos =======================
 * Free disk space under / :
 Filesystem      Size  Used Avail Use% Mounted on
-/dev/loop0       60G   39G   21G  65% /
+/dev/loop0       60G   41G   19G  69% /
```

Comment:
we should plan to remove a couple of tools, most likely older Android build tools as free disk just went under 20GB